test/integration/basic_test.go

```diff
@@ -66,6 +66,16 @@ func (m *Minishift) shouldHaveAValidIPAddress() error {
 	return nil
 }
 
+func shouldHaveSudoerAccess() error {
+	ocRunner := util.NewOcRunner()
+	cmdArgs := []string{"--as", "system:admin", "get", "node"}
+	cmdOut, _ := ocRunner.RunCommand(cmdArgs)
+	if !strings.Contains(string(cmdOut), "NAME") {
+		return fmt.Errorf("Not able to set developers as sudoers: %s", string(cmdOut))
```

Comment:
maybe I am not following your intention here. likely due to the fact of the error message.

Does it set the access level, or perform an action as a certain user?